blackboxopt/optimizers/space_filling.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2020 - for information on the respective copyright owner
+# see the NOTICE file and/or the repository https://github.com/boschresearch/blackboxopt
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import List
+
+try:
+    import numpy as np
+    from scipy.stats.qmc import Sobol
+except ImportError as e:
+    raise ImportError(
+        "Unable to import BOHB optimizer specific dependencies. "
```

Comment:
This should probably read:
```suggestion
        "Unable to import SpaceFilling optimizer specific dependencies. "
```